components/button.css

```diff
@@ -0,0 +1,187 @@
+.btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  outline: none;
+  white-space: nowrap;
+}
+
+.btn:focus {
+  outline: 2px double var(--bg-primary);
+  outline-offset: 2px;
+}
+
+.btn i {
+  margin: 0 0.4rem;
+}
+
+/* buttons */
+.btn:not(.btn-icon) {
+  border: 1px solid transparent;
+  border-radius: 0.475rem;
+  padding: 0.7rem 0.8rem;
+  color: var(--white-text);
+  transition: background-color 0.2s linear;
+}
+
+.btn[disabled] {
+  opacity: 0.5;
+  pointer-events: none;
+}
+
+.btn-primary {
+  background-color: var(--bg-primary);
+}
+
+.btn-primary:hover {
+  background-color: #013f9b;
+}
+
+.btn.btn-primary-outline {
+  border: 1px solid var(--bg-primary);
+  color: black;
+}
+
+.btn-primary-outline:hover {
+  background-color: var(--bg-primary);
+  color: var(--white-text);
+}
+
+.btn.btn-secondary {
+  background-color: var(--bg-secondary);
+  color: black;
+}
+
+.btn-secondary:hover {
+  background-color: #6a8686;
```

Comment:
Don't hard code the color value used as a variable.

Reply:
This value is the darker version of secondary color and it is supposed to be used only in one place, so didn't find any need to add it as a variable.